.travis.yml

```diff
@@ -15,7 +15,7 @@ env:
         - FOLDERNAME='packagename'
         - PYTHON_VERSION=3.7
         - CONDA_DEPENDENCIES='cython astropy sphinx'
-        - PIP_DEPENDENCIES='cookiecutter gitpython pytest-astropy sphinx-astropy'
+        - PIP_DEPENDENCIES='cookiecutter gitpython pytest-astropy pytest-astropy-header sphinx-astropy'
```

Comment:
pytest-astropy should contain pytest-astropy-header

Reply:
I added it back in. For some reason, removing it caused Travis to fail for all but the last job. :woman_shrugging: